book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rentals(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] A book has many rentals, while a rental belongs to a book and also belongs to a person. Please initialize the Rental here (`Rental.new`) while passing in the book itself as a parameter to create the new rental achieving the 'has-many' association. 
```ruby
def add_rental(person, date)
  Rental.new(date, self, person)
end
```
#### 🔔 Kindly do the same for the `Person` class
Afterward, the constructor of `Rental` should be modified so `Book` and `Person` are set in it to break the `many-to-many` relationship.